import/prefer-default-export should ignore type for flowtype

I have a file with this:

``` js
// @flow
export type CommentType = {
  mobile_image_thumbnail: string,
  author_name: string,
  preview_text: string,
  updated_at: string,
}
```

I get this error:

app/bundles/comments/types/index.js
  2:1  error  Prefer default export  import/prefer-default-export

[See doc note](https://flowtype.org/blog/2015/02/18/Import-Types.html):

```
Note that we only support the explicit named-export statements for now (i.e. export type UserID = number;). In a future version we can add support for latent named-export statements (i.e. type UserID = number; export {UserID};) and default type exports (i.e. export default type MyType = ... ;)…but for now these forms aren’t yet supported for type exports.
```

I just checked and `export default type` is not supported in 0.27.

Any chance of ignoring `export type` as an option to this rule?

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,7 @@ This change log adheres to standards from [Keep a CHANGELOG](http://keepachangel
 
 ## [Unreleased]
 ### Fixed
+- [`prefer-default-export`] handles flow `export type` ([#484])
 - [`prefer-default-export`] handles re-exported default exports ([#609])
 - Fix crash when using `newline-after-import` with decorators ([#592])
 - Properly report `newline-after-import` when next line is a decorator

--- a/src/rules/prefer-default-export.js
+++ b/src/rules/prefer-default-export.js
@@ -42,6 +42,9 @@ module.exports = {
         // if there are specifiers, node.declaration should be null
         if (!node.declaration) return
 
+        // don't count flow types exports
+        if (node.exportKind === 'type') return
+
         if (node.declaration.declarations) {
           node.declaration.declarations.forEach(function(declaration) {
             captureDeclaration(declaration.id)

--- a/tests/src/rules/prefer-default-export.js
+++ b/tests/src/rules/prefer-default-export.js
@@ -66,6 +66,11 @@ ruleTester.run('prefer-default-export', rule, {
         import * as foo from './foo';`,
       }),
 
+    test({
+      code: `export type UserId = number;`,
+      parser: 'babel-eslint',
+      }),
+
     // ...SYNTAX_CASES,
   ],
   invalid: [

